CONTRIBUTING.md

```diff
@@ -14,7 +14,7 @@ Documentation
 -------------
 
 If you contribute anything that changes the behavior of the
-application, document it in the [README](https://github.com/Comcast/scytale/blob/master/README.md) or [wiki](https://github.com/Comcast/scytale/wiki)! This includes new features, additional variants of behavior and breaking changes.
+application, document it in the [README](https://github.com/Comcast/scytale/blob/main/README.md) or [wiki](https://github.com/Comcast/scytale/wiki)! This includes new features, additional variants of behavior and breaking changes.
```

Comment:
This is a sign that the contributing.md is old; can you update it to match this?
https://github.com/xmidt-org/.go-template/blob/main/CONTRIBUTING.md